src/cascadia/TerminalApp/CascadiaSettings.h

```diff
@@ -20,6 +20,8 @@ Author(s):
 #include "GlobalAppSettings.h"
 #include "Profile.h"
 
+extern GUID AzureConnectionType;

```

Comment:
I'm concerned about this being extern as that can be a bit fragile. Can you explain why that's necessary here?

Reply:
Will change it to a static constexpr in the header file